pkg/retry/retry.go

```diff
@@ -88,9 +88,8 @@ func (s *Server) handleRetry() http.Handler {
 		// for each run track the total number of events retrieve and failure events
 		var totalEventCount int
 		var failedEventCount int
-		var newCheckpoint *int64
+		var newCheckpoint string
```

Comment:
Whats the reason for the type change?

Reply:
we store checkpoints as string, it felt more consistent.

Reply:
It seems in BQ the checkpoint is a string and `lastCheckpoint` is also a string. So sooner or later we need to convert the delivery id to a string.

Reply:
right, it might be confusing referring to checkpoints (newCheckpoint and lastCheckpoint) as two different types.